start_bs.py

```diff
@@ -172,9 +172,9 @@ class SampleXYZ(Device):
     raster_flyer = MXRasterFlyer(vector, zebra, detector)
 
     from nyxtools.isara_robot import IsaraRobotDevice
-    from denso_robot import DensoRobot
-    denso_ophyd_robot = IsaraRobotDevice("XF19IDC-ES{Rbt:1}", name="robot")
-    robot = DensoRobot(denso_ophyd_robot) # DensoRobot is the robot_lib API-compatible object
+    from denso_robot import OphydRobot
+    ophyd_robot = IsaraRobotDevice("XF19IDC-ES{Rbt:1}", name="robot")
+    robot = OphydRobot(ophyd_robot) # DensoRobot is the robot_lib API-compatible object
```

Comment:
you probably want to change of the class name in the comment

Reply:
this was done in d8ce8bf